commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPIConfig.java

```diff
@@ -90,6 +91,22 @@ public Impl useLatestNMSVersion(boolean value) {
 		return instance();
 	}
 
+	/**
+	 * Sets whether the CommandAPI should load a (potentially unsupported) NMS version
+	 * when updating to a minor release of Minecraft, for example this setting would
```

Comment:
Should this be a new sentence?

```suggestion
	 * when updating to a minor release of Minecraft. For example, this setting would
```

Also, I like the example of `this setting can allow updating to 1.21.2 from 1.21.1 but doesn't allow updating to 1.22 from 1.21.2` from the docs. I think it makes the meaning of "minor release" very clear given that Minecraft doesn't quite follow the common SemVer logic of `major.minor.patch` (1.21 to 1.22 might seem like a change in the minor version, and hence be confused as a minor update). Perhaps that could be incorporated in these javadocs?